lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -786,7 +786,7 @@ def test_render_html_view_with_preview_mode_when_user_already_has_cert(self):
         # user has already has certificate generated for 'honor' mode
         # so let's try to preview in 'verified' mode.
         response = self.client.get(test_url + '?preview=verified')
-        self.assertNotIn(self.course.display_name, response.content)
+        self.assertNotIn(self.course.display_name.encode('utf-8'), response.content)
```

Comment:
Yes.  lxml's unicode handling doesn't map cleanly from py2 to py3, so I had to clean up our use of it a little, but that impacted the return types in a couple places.  I feel pretty comfortable saying that `display_name` shouldn't be a bag of bytes.